feature-utils/poly-look/src/react-components/banners/notificationBanner.css

```diff
@@ -0,0 +1,29 @@
+.notification-banner {
+    height: 56px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: calc(var(--gdu) * 4);
+    margin: calc(var(--gdu) * -6) calc(var(--gdu) * -4) calc(var(--gdu) * 6) calc(var(--gdu) * -4);
+    box-sizing: border-box;
+}
+
+.standard {
+    background-color: var(--color-gray-100);
+    color: var(--color-gray-900);
+}
+
+.error {
```

Comment:
`standard` is probably OK, but this and below are too generic to use them only here, right?

Reply:
I added the container class before each one, so even though the className could be generic, it will only affect this component in this context.